libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -555,6 +555,7 @@ dictionary SwapInfo {
     sequence<string> confirmed_tx_ids;
     i64 min_allowed_deposit;
     i64 max_allowed_deposit;
+    i64 max_allowed_deposit_abs;
```

Comment:
NIT: rename to `max_swapper_payable` ? because it is actually the maximum the swapper is willing to pay.

Reply:
Done: a29023c2e97b9c13a86c04944e04a7b021ee6eea